CHANGELOG.md

```diff
@@ -5,6 +5,17 @@ All notable changes to the [Nucleus Python Client](https://github.com/scaleapi/n
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.17.2](https://github.com/scaleapi/nucleus-python-client/releases/tag/v0.17.2) - 2024-02-28
+
+### Modified
+- In `Dataset.create_slice`, the `reference_ids` parameter is now optional. If left unspecified, it will create an empty slice
+
+
+## [0.17.1](https://github.com/scaleapi/nucleus-python-client/releases/tag/v0.17.1) - 2024-02-14
+
+No major changes
+
+
```

Comment:
Weird - I added a changelog in https://github.com/scaleapi/nucleus-python-client/pull/431/files

```
## [0.17.1] - 2024-02-22

### Added
- Environment variable `NUCLEUS_SKIP_SSL_VERIFY` to skip SSL verification on requests
```